Base/QTApp/qSlicerMainWindow.cxx

```diff
@@ -630,7 +630,7 @@ bool qSlicerMainWindowPrivate::confirmCloseApplication()
   bool close = false;
   if (!question.isEmpty())
     {
-    QMessageBox* messageBox = new QMessageBox(QMessageBox::Warning, qSlicerMainWindow::tr("Save before exit?"), question, QMessageBox::NoButton);
+    QMessageBox* messageBox = new QMessageBox(QMessageBox::Warning, qSlicerMainWindow::tr("Save before exit?"), question, QMessageBox::NoButton, q_ptr);
```

Comment:
Is there any specific reason you used `q_ptr` instead of `q`? A few lines below we already use `q` in a very similar role.